adjust

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python3
+
+# Adapted from https://github.com/opsani/servo-statestore/blob/master/adjust
+
+import math
+import os
+import sys
+
+# if this is running in the pytest suite, symlink destination names have been adjusted to make importing easier
+if 'pytest' in sys.modules:
```

Comment:
Can you change this so that the test has all the logic on how to deal with imports rather than the actual driver.